Models/Shape.cs

```diff
@@ -11,8 +11,36 @@ namespace DrawMat.Models;
 
 public abstract class ShapeBase
 {
-    public abstract Control ToControl();
+    public Rect BoundingBox { get; set; } = new();
     public Point Origin { get; set; } = new();
+    
```

Comment:
By the way, there's trailing whitespace here. `git show` highlights it, but GitHub doesn't. Probably because people who're OCD about whitespace are vastly outnumbered by those who don't even notice it.